client.js

```diff
@@ -72,13 +92,12 @@ function insert({ _, ...rest }) {
     accountId: rest.accoutId || 0
   };
   console.log(user);
-  client.insert(user, (err, user) => {
-    if (err) {
-      console.log(err);
-    } else {
-      console.log(user);
-    }
-  });
+  await client.insert({ user, edit: rest.edit });
+  if (rest.edit) {
+    console.log("User editted successfully.");
+  } else {
+    console.log("User inserted successfully.");
```

Comment:
I can insert the same user multiple times, and the subsequent requests appear to succeed. When I then fetch, I can see that nothing was overwritten (unless I use update), but we should return an error code and let the user know that the ID was already taken.